lab-papers-please/csharp-classification/src/csharp/Program.cs

```diff
@@ -0,0 +1,84 @@
+﻿using System;
+using System.Text.Json;
+using System.Text.Json.Nodes;
+
+namespace Csharp
+{
+    class Program
+    {
+        static void Main()
+        {
+            //var inputFile = "..\\resources\\test-input.json";
+            string baseDir = AppContext.BaseDirectory;
+
+            // Adjust the path relative to the base directory where the app is running
+            var inputFile = Path.Combine(baseDir, "..", "..", "..", "..", "resources", "input", "test-input.json");
```

Comment:
Is there a way not to use relative path, but project path? 
Relative path is not recommended since a change in path would mean a change in code, is there a way to reference the file by a designated resource directory? Similar to java solution.

![Screenshot_20241013_143047_GitHub.jpg](https://github.com/user-attachments/assets/2230b36d-68ea-4b14-a0f2-a6f24e10ee41)

Reply:
From what I have documented myself, this is how it should work, we have to go up several folders from the location of the executable that's in bin\Debug\net8.0, so I either go up a few levels and set a path relative to it, or I set an absolute path for the JSON file.